File: copy-image.py
import shutil
import os

src = r'e:\WEBSITE CREAT for MY BUSSINESS\Resume(CV) & Cover letter\pictur and video logo\bangladesh_cv_template.png'
dst = r'e:\WEBSITE CREAT for MY BUSSINESS\Resume(CV) & Cover letter\public\pictur and video logo\bangladesh_cv_template.png'

try:
    if os.path.exists(src):
        # Ensure destination directory exists
        os.makedirs(os.path.dirname(dst), exist_ok=True)
        shutil.copy2(src, dst)
        print("SUCCESS: copied image to public directory:", dst)
    else:
        print("ERROR: Source file does not exist:", src)
except Exception as e:
    print("ERROR:", e)

File: copy-image.js
import fs from 'fs';
import path from 'path';

const source = "pictur and video logo/bangladesh_cv_template.png";
const dest = "public/pictur and video logo/bangladesh_cv_template.png";

try {
  if (fs.existsSync(source)) {
    // Ensure destination directory exists
    const destDir = path.dirname(dest);
    if (!fs.existsSync(destDir)) {
      fs.mkdirSync(destDir, { recursive: true });
    }
    fs.copyFileSync(source, dest);
    console.log("SUCCESS: copied image to public directory:", dest);
  } else {
    console.log("Source does not exist: " + source);
  }
} catch (err) {
  console.error("ERROR: ", err);
}

